src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+        '200':
+          description: ''
+  '/octoprint-emulator/api/files/local':
+    post:
+      summary: 'Upload gcode'
+      operationId: server.routes.octoprintemulator.upload
+      tags:
+        - octoprint-emulator
+      responses:
+        '200':
+          description: ''
+  '/organizations':
+    post:
+      summary: 'Create organization'
+      operationId: server.routes.organizations.create_organization
+      tags:
+        - organizations
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                name:
+                  type: string
+      responses:
+        '201':
+          description: 'Created'
+    get:
+      summary: 'List organizations'
+      operationId: server.routes.organizations.list_organizations
+      tags:
+        - organizations
+      responses:
+        '200':
+          description: 'List of organizations'
+  '/organizations/{org_uuid}':
+    patch:
+      summary: 'Reanme orgaization'
+      operationId: server.routes.organizations.update_organization
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                name:
+                  type: string
+      tags:
+        - organizations
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'Renamed'
+  '/organizations/{org_uuid}/printjobs':
+    post:
+      summary: 'Start printing gcode on a printer'
+      operationId: server.routes.printjobs.printjob_create
+      tags:
+        - printjobs
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                gcode:
+                  type: string
+                  format: uuid
+                printer:
+                  type: string
+                  format: uuid
+      responses:
+        '201':
+          description: 'Printjob scheduled'
+        '409':
+          description: Could ot schedule a printjob
+    get:
+      summary: 'List printjobs'
+      operationId: server.routes.printjobs.printjobs_list
+      tags:
+        - printjobs
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+      responses:
+        '200':
+          description: 'List of printjobs'
+  '/organizations/{org_uuid}/printjobs/{printjob_uuid}':
+    get:
+      summary: 'Get printjob detail'
+      operationId: server.routes.printjobs.printjob_detail
+      tags:
+        - printjobs
+      parameters:
+        - $ref: '#/components/parameters/organization_uuid_param'
+        - $ref: '#/components/parameters/printjob_uuid_param'
+      responses:
+        '200':
+          description: 'Printjob detail'
+  '/organizations/{org_uuid}/tasks':
+    post:
+      summary: 'Schedule a task for organization'
```

Comment:
A better description what a task actually is would be helpful here

Reply:
Added better description